data/src/main/java/com/m3u/data/parser/epg/EpgData.kt

```diff
@@ -30,6 +30,11 @@ data class EpgProgramme(
     val title: String? = null,
     val desc: String? = null,
     val icon: String? = null,
+    val isNew: Boolean,
+    val isNewTag: Boolean,
+    val isLive: Boolean,
```

Comment:
add default values

Reply:
are default values necessary? it already loads the 0 if not true. but i added

Reply:
add it is better